src/javascript/SelectorTypes/SystemName/SystemName.jsx

```diff
@@ -3,28 +3,55 @@ import React from 'react';
 import {FieldPropTypes} from '~/FormDefinitions/FormData.proptypes';
 import Text from '~/SelectorTypes/Text';
 import {Constants} from '~/ContentEditor.constants';
+import {Button, Copy} from '@jahia/moonstone';
+import {withStyles} from '@material-ui/core';
+import {useTranslation} from 'react-i18next';
+
+const styles = theme => ({
```

Comment:
Please use module css instead of JSX here.
This will help also to use Moonstone variables instead of old material ui override theme.